tests/qa/qatests

```diff
@@ -28,7 +28,10 @@ RS_PLATFORMS = {
     'cenos7': {
         'platform': 'Linux-rhel7',
         'env': 'rhel7.7-amd64-aws' },
-    'cenos8': {
+    'centos8': {
```

Comment:
why keep centos8 if it is replaced??

Reply:
It's not being replaced. I think once they fix it, it will be back. Note that 'centos8' is not just CentOS 8 literally, but the class of systems which are compatible with RHEL8. The point is that some systems, like Oracle Linux and Amazon Linux are not exactly 100% compatible with it (for instance, names of some packages may be different), so we keep them a little bit apart until we know them better. They do qualify for `redhat_compat` in `system-setup.py` though.